.github/workflows/release.yaml

```diff
@@ -36,35 +38,19 @@ jobs:
         with:
           latest-version: ${{ steps.tag_version.outputs.new_tag }}
           release-notes: ${{ steps.tag_version.outputs.changelog }}
-      - name: Commit bumpversion
-        id: bumpversion
-        if: steps.tag_version.outputs.new_version
-        uses: stefanzweifel/git-auto-commit-action@v4.14.1
-        with:
-          branch: ${{ github.ref }}
-          commit_message: "chore: preparing release ${{ steps.tag_version.outputs.new_version }}"
-          file_pattern: CHANGELOG.md setup.cfg tutoraspects/*
-  release:
-    needs: bumpversion
-    if: needs.bumpversion.outputs.version
-    runs-on: ubuntu-latest
-    outputs:
-      tag: ${{ steps.tag_version.outputs.new_tag }}
-      changelog: ${{ steps.tag_version.outputs.changelog }}
-    steps:
-      - uses: actions/checkout@v3
-      - name: Create tag
-        id: tag_version
-        uses: mathieudutour/github-tag-action@v6.0
+      - name: Push branch
+        run: |
+          git checkout -b bot/v${{ steps.tag_version.outputs.new_version }}
+          git push origin bot/v${{ steps.tag_version.outputs.new_version }}
+      - uses: tibdex/github-app-token@v1
+        id: generate-token
         with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          commit_sha: ${{ needs.bumpversion.outputs.bump_commit_sha }}
-          default_bump: false
-          default_prerelease_bump: false
-      - name: Create a GitHub release
-        if: steps.tag_version.outputs.new_tag
-        uses: ncipollo/release-action@v1
+          app_id: ${{ secrets.APP_ID }}
```

Comment:
I think we can scrap the app and use the env:

`GITHUB_TOKEN: ${{ secrets.SEMANTIC_RELEASE_GITHUB_TOKEN }}`